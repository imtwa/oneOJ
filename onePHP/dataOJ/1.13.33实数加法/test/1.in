25683827882.3787214462342733184724211466382411132585545828722552874786227534713784536644224
47648688437.1238627246851465811264718154163154772853388162112414334552541118425313276262565
