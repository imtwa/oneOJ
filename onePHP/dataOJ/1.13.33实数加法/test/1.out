73332516319.5025841709194198995988929620545565905438933990834967209338768653139097812906789